receiver.go

```diff
@@ -52,7 +52,7 @@ type (
 		prefetch           uint32
 		DefaultDisposition DispositionAction
 		Closed             bool
-		doneRefreshingAuth func()
+		doneRefreshingAuth func() <-chan struct{}
```

Comment:
I think it would be clearer to call this `stopRefreshingAuth()`

Reply:
Agreed, updated.